### ISSUE TITLE
Autostart didn't work for LinuxGSM tf2server on us1, us3, as1

### ISSUE THREAD

**Issue by JONTTM** [leighmacdonald/uncletopia#65]
Both servers restarted today and tf2server was not started on boot like it should. Investigate systemd.

**Comment by JONTTM** [comment]
as1 is now offline, possibly same issue

**Comment by JONTTM** [comment]
After investigating possible causes, the most likely cause was `tf2server monitor` 

systemd indicates it started tf2server after system reboot as it should, but the log also displays tf2server shutting down about a minute later. I expect monitor bugged out and tried to restart the server while it was starting.

monitor is supposed to not do anything if the server is stopped or starting:
executing `tf2server start` yields:
```
tf2server@dane-as1:~$ ./tf2server monitor
[ ERROR ] Monitoring tf2server: Checking lockfile: No lockfile found: ERROR
* Start tf2server to run monitor.
tf2server@dane-as1:~$ ./tf2server monitor
[  OK  ] Monitoring tf2server: Checking session: OK
[  OK  ] Monitoring tf2server: Querying port: gsquery: 0.0.0.0:27015 : 0/1: DELAY
tf2server@dane-as1:~$ ./tf2server monitor
[  OK  ] Monitoring tf2server: Checking session: OK
[  OK  ] Monitoring tf2server: Querying port: gsquery: 0.0.0.0:27015 : 0/1: DELAY
tf2server@dane-as1:~$ ./tf2server monitor
[  OK  ] Monitoring tf2server: Checking session: OK
[  OK  ] Monitoring tf2server: Querying port: gsquery: 10.148.0.2:27015 : 0/1: OK
```
ERROR indicates the server is stopped
DELAY indicates the server is still starting (don't query port)
OK indicates the port was queried and answered (no need to restart server)

I don't know why this happened, but if it continues to happen in the future my first idea will be to widen the cron delay on monitor (currently */1 minute). We used to have this set to */5 minute but this was changed a while ago. It's possible that changing this to */1 minute caused this to happen, although a better solution would be to prevent tf2server monitor from executing at all around restart time (maybe from 05:00 to 05:05 server time?). Closing for now but will look into a possible fix if it continues to happen.
